packages/slider/src/ux-slider.css

```diff
@@ -0,0 +1,76 @@
+ux-slider {
```

Comment:
I see an issue with the way the CSS classes are used here. We prefer to use BEM styling for our CSS classes. You can read up on that here!

http://getbem.com/introduction/

Take a look at this file for the UX Switch for an example of this being used within Aurelia UX.

https://github.com/aurelia/ux/blob/master/packages/switch/src/ux-switch.css

Reply:
neat!  never heard of BEM before but I like it.  I'll fix it up and let you know

Reply:
let me know if i messed up on this.  it seemed pretty straightforward but it wouldnt be the first time I've thought something was simpler than it actually was :P